src/applications/simple-forms/20-10206/pages/disabilityExamDetails.js

```diff
@@ -19,6 +21,13 @@ export default {
     disabilityExams: {
       'ui:options': {
         itemName: 'exam date',
+        itemAriaLabel: formData => {
+          const friendlyDateString = moment(
```

Comment:
if formData.disabilityExamDate is null, will this handle this OK? maybe we could return "Exam date not specified?" or something?

Reply:
tested - current behavior seems OK - it will say invalid date